lib/opal/cli_runners/cmd.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+require 'shellwords'
+
+module Opal
+  module CliRunners
+    class Cmd
+      def initialize(options, name, env, *cmd)
+        @output = options.fetch(:output, $stdout)
+        @name = name
+        @env = env
+        @cmd = *cmd
```

Comment:
Is `*` required here?

Reply:
probably not, I'm not familiar with the splat operator